examples/flask/README.md

```diff
@@ -0,0 +1,53 @@
+# Flask Sample Application for Pybrake
+
+## About the application:
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time. 
+2. `/locations` - returns list of available locations. 
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API:
+
+1. Install the dependencies for the application
+
+```bash
+pip3 install -r requirements.txt
```

Comment:
Why do we require `pip3` instead of `pip`? My impression was that we'd want `pip`, since that's the default command.

Reply:
`pip3` and `pip` both are same command. If python v2 and v3 both install in system at that time pip3 refers to v3. But as v2 is totally deprecated. I'll replace it with `pip`.